napari/_qt/dialogs/_tests/test_installer_process.py

```diff
@@ -0,0 +1,217 @@
+import re
+import sys
+import time
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+import pytest
+from qtpy.QtCore import QProcessEnvironment
+
+from napari._qt.dialogs.qt_package_installer import (
+    AbstractInstallerTool,
+    CondaInstallerTool,
+    InstallerQueue,
+    InstallerTools,
+    PipInstallerTool,
+)
+
+if TYPE_CHECKING:
+    from virtualenv.run import Session
+
+
+@pytest.fixture
+def tmp_virtualenv(tmp_path) -> 'Session':
+    virtualenv = pytest.importorskip('virtualenv')
+
+    cmd = [str(tmp_path), '--no-setuptools', '--no-wheel', '--activators', '']
+    return virtualenv.cli_run(cmd)
+
+
+@pytest.fixture
+def tmp_conda_env(tmp_path):
```

Comment:
Do these environment fixtures get cleaned up?

Reply:
Eventually, yes. Three or so are left around for some rounds to aid debugging, but it ends up being deleted.

See https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html#the-default-base-temporary-directory

Reply:
Cool, thank you!